src/app.tsx

```diff
@@ -71,6 +71,11 @@ const injectCSS = () => {
 const tagItems = () => {
   const yourEpisodesInSidebar = document.querySelector('a[href="/collection/episodes"]')?.parentElement;
   if (yourEpisodesInSidebar) yourEpisodesInSidebar.classList.add('podcast-item');
+  
+  // remove podcast shortcuts on home
+  document.querySelectorAll('.view-homeShortcutsGrid-name a[href^="/episode"]').forEach((selected) => {
+    selected.parentNode.parentNode.parentNode.parentNode.parentNode.classList.add('podcast-item');
+  });
```

Comment:
So I have the `.view-homeShortcutsGrid-grid` elements in my DOM now. I don't have a podcast item, but your code looks sound. I just cleaned it up a little. Was going to push up to this PR but my VS Code is being weird and creating a new branch on the main repo when I try and push, so I'll just put this here. 

```suggestion

  // Remove podcast shortcuts on home page
  const podcastShortcutsOnHomepage = document.querySelectorAll('.view-homeShortcutsGrid-name a[href^="/episode"]');
  podcastShortcutsOnHomepage.forEach((itemLink) => {
    const item = itemLink.closest('.view-homeShortcutsGrid-shortcut');
    if (item) item.classList.add('podcast-item');
  });
```